### PR TITLE
[MERGE][MOV] sale: split and reorganize python files

### PR DESCRIPTION
**Pull request by Feyensv**
The `sale` code is going to experience some refactorings/improvements soon, and in preparation of those tasks, we believed it was time to reorder the python files.  Splitting files according to python classes, and using targeted names as file names will improve the clarity and maintainability of the module, easing the future work.

To conserve the blame history of the splitted files, the file split are done in 2+ commits.

* `account_move.py` & `account_invoice.py` -> `account.move{.line}.py`
* `payment.py` -> `payment_acquirer.py` & `payment_transaction.py`
* `sale.py` -> `sale.order{.line}.py`
* `utm.py` -> `utm_campaign.py`

Moved the `mail_compose_message.py` file to the `wizard` folder, since it targets a wizard.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Feyensv** [comment]
@robodoo override=ci/style

**Comment by Feyensv** [comment]
@robodoo rebase-merge

**Comment by tivisse** [comment]
LGTM.

File: addons/sale/models/__init__.py [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
+from . import sale_order
+from . import sale_order_line
 from . import res_config_settings
 from . import sales_team
-from . import payment
-from . import utm
-from . import mail_compose_message
+from . import payment_acquirer
+from . import payment_transaction
+from . import utm_campaign
```
Comment (AntoineVDV) [inline addons/sale/models/__init__.py:17]: alphanum order

**Review by AntoineVDV — APPROVED** [review]

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**